lib/spack/spack/build_environment.py

```diff
@@ -1266,17 +1301,20 @@ def _setup_pkg_and_run(
             input_pipe.close()
 
 
-def start_build_process(pkg, function, kwargs):
+def start_build_process(
+    pkg: "spack.package_base.PackageBase", function: Callable, kwargs: Dict[str, Any]
+):
     """Create a child process to do part of a spack build.
 
     Args:
 
-        pkg (spack.package_base.PackageBase): package whose environment we should set up the
-            child process for.
-        function (typing.Callable): argless function to run in the child
+        pkg: package whose environment we should set up the
+             child process for.
+        function: argless function to run in the child
             process.
+        kwargs: additional keyword arguments to pass to ``function()``
 
-    Usage::
+    Usage:
```

Comment:
```suggestion
    Usage::
```

The double colon here was intentional -- it starts a literal block (i.e. for code):
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks